fourier/fourier.go

```diff
@@ -40,15 +40,15 @@ func (t *FFT) Reset(n int) {
 	fftpack.Rffti(n, t.work, t.ifac[:])
 }
 
-// FFT computes the Fourier coefficients of the input sequence, seq,
+// Coefficients computes the Fourier coefficients of the input sequence, seq,
 // placing the result in dst and returning it. This transform is
-// unnormalized; a call to FFT followed by a call of IFFT will
+// unnormalized; a call to Coefficients followed by a call of Sequence will
 // multiply the input sequence by the length of the sequence.
 //
-// If the length of seq is not t.Len(), FFT will panic.
+// If the length of seq is not t.Len(), Coefficients will panic.
 // If dst is nil, a new slice is allocated and returned. If dst is not nil and
-// the length of dst does not equal t.Len()/2+1, FFT will panic.
-func (t *FFT) FFT(dst []complex128, seq []float64) []complex128 {
+// the length of dst does not equal t.Len()/2+1, Coefficients will panic.
+func (t *FFT) Coefficients(dst []complex128, seq []float64) []complex128 {
```

Comment:
I didn't make it in time for the initial PR, so I'll comment here. I think the type would be better named `DFT` because it computes the Discrete Fourier Transform (the problem being solved) while the Fast Fourier Transform is the (family of) algorithm that computes it (implementation detail that solves the problem). It would be also more consistent with the transforms introduced in this PR. Although I do know that DFT and FFT are often used interchangeably.

Reply:
Unfortunately discoverability will likely trump correctness.